content/topics/sharing-code-workflows.md

```diff
@@ -0,0 +1,84 @@
+
+<a class="link-teal" href="https://www.software.ac.uk/blog/why-should-you-care-about-reproducible-code-and-how-get-started" target="_blank"> <b>Learn more about practical benefits and how to get started <i class="bi bi-box-arrow-up-right"></i></b></a>
+
+<a class="link-teal" href="https://urn.kb.se/resolve?urn=urn:nbn:se:kb:publ-738" target="_blank"><b>Download and read the National Library of Sweden’s national guidelines <i class="bi bi-box-arrow-up-right"></i></b></a>
+
+## How to share code & workflows?
+
+How you share code and workflows depends on your goals—whether you seek transparency, collaboration, or community contributions. Keep in mind that your objectives may evolve over time, and every step towards FAIR practices adds value.  
+
+<a class="link-teal" href="https://github.com/ScilifelabDataCentre/open-science-checklists/blob/main/open_software_checklist.md" target="_blank"> <b>Consult the SciLifeLab Open Science Software Checklist for comprehensive guidance<i class="bi bi-box-arrow-up-right"></i></b></a>
+
+### Where can code and workflows be shared?
+
+To ensure transparency, make a static copy of your code or workflow available for long-term access, independent of any research paper. This should be accompanied by a README file with a clear description, usage instructions, and license(s). Collaborative version control platforms, such as GitHub, are ideal for both sharing and ongoing development. Use tags or releases to reference specific versions of your code.
+
+To enhance FAIRness, consider obtaining a persistent identifier by publishing your code or workflow in a research output repository, such as [Zenodo](https://zenodo.org/) or the [SciLifeLab Data Repository](https://figshare.scilifelab.se/). This can be achieved by linking your GitHub repository or uploading files directly.
+
+If your code or workflow is intended for reuse, distribute it through a package manager (e.g., [BioConda](https://bioconda.github.io/) or [CRAN](https://cran.r-project.org/)). For workflows, consider joining communities like [nf-core](https://nf-co.re/) or [Galaxy Europe](https://galaxyproject.org/eu/), and register your workflow on platforms such as [WorkflowHub](https://workflowhub.eu/). These steps help others build upon your work.
+
+For software, consider publishing in a dedicated software journal (e.g., [Journal of Open Source Software](https://joss.theoj.org/)) or as a software article in a life sciences journal (e.g., [PLOS Computational Biology](https://collections.plos.org/collection/software/) or [BMC Bioinformatics](https://bmcbioinformatics.biomedcentral.com/submission-guidelines/preparing-your-manuscript/software-article)).
+
+### Tips & Tricks
+
+- Start from a template or "cookie cutter" to streamline setup and follow best practices. 
+- Write readable code by using meaningful variable names, logical structure, and clear comments. Some languages offer packages/tools that lint and format code for consistency. 
```

Comment:
```suggestion
- Write readable code by using meaningful variable names, a logical structure, and clear comments. Some coding languages offer packages/tools that lint and format code for consistency. 
```

Reply:
linting can also be achieved by actions (e.g. markdown linting action on GitHub)